csharp/ql/src/utils/model-generator/internal/ModelGeneratorUtilsSpecific.qll

```diff
@@ -63,3 +68,11 @@ string returnNodeAsOutput(ReturnNodeExt node) {
       result = "Argument[Qualifier]"
     )
 }
+
+module DataFlowPrivate {
```

Comment:
Can be removed when renaming `Dfp` to `DataFlowPrivate`

Reply:
Same comment as above